src/api/Versions.js

```diff
@@ -249,7 +250,7 @@ class Versions extends OffsetBasedAPI {
             data: {
                 data: {
                     id: versionId,
-                    type: 'file_version',
```

Comment:
I don't think this is desired, since versions are used outside of the Activity Feed experience. The API layer and types should drive the experiences, not the other way around.